.bazelci/config.yaml

```diff
@@ -43,6 +47,8 @@ tasks:
     working_directory: examples
     build_targets:
       - "//cmake_android:app"
+      - "//boost:boost_regex"
+      #- "//boost:boost_filesystem"
```

Comment:
What happened with this test?

Reply:
It was passing invalid arguments to the build; because on macOS clang masquerades as g++ by default then boost's build system is picking the wrong compiler flags. As a result this target fails to build on macOS